firecares/firecares_core/context_processors.py

```diff
@@ -9,3 +10,19 @@ def third_party_tracking_ids(request):
     return {
         'google_analytics_tracking_id': settings.GOOGLE_ANALYTICS_TRACKING_ID,
     }
+
+
+def fire_department_search(request):
+    """
+    Required context variables for fire department search.
+    """
+    context = {}
+    score_metrics = FireDepartment.objects.all().aggregate(Max('dist_model_score'), Min('dist_model_score'))
+    context['dist_max'] = score_metrics['dist_model_score__max']
+    context['dist_min'] = score_metrics['dist_model_score__min']
+
+    population_metrics = FireDepartment.objects.all().aggregate(Max('population'), Min('population'))
```

Comment:
1. Can `Max('population'), Min('population')` be added to the `score_metrics` query instead of querying the db again?

2. Use Django's (cache framework)[https://docs.djangoproject.com/en/1.9/topics/cache/#basic-usage] to cache this request so we're not re-querying the DB for the same information in every request.  I'd say we store the results for at least a day, we can manually bust when updating the underlying data.